src/djangosaml2_spid/views.py

```diff
@@ -362,108 +329,99 @@ def spid_sp_metadata(conf):
         assertion_consumer_service.index = str(cnt)
         cnt += 1
 
-    # nameformat patch... non proprio standard
+    # nameformat patch
     for reqattr in metadata.spsso_descriptor.attribute_consuming_service[0].requested_attribute:
-        reqattr.name_format = None #"urn:oasis:names:tc:SAML:2.0:attrname-format:basic"
-        # reqattr.is_required = None
+        reqattr.name_format = None  # "urn:oasis:names:tc:SAML:2.0:attrname-format:basic"
         reqattr.friendly_name = None
 
-    # remove unecessary encryption and digest algs
-    # supported_algs = ['http://www.w3.org/2009/xmldsig11#dsa-sha256',
-    #                   'http://www.w3.org/2001/04/xmldsig-more#rsa-sha256']
-    # new_list = []
-    # for alg in metadata.extensions.extension_elements:
-        # if alg.attributes.get('Algorithm') in supported_algs:
-            # new_list.append(alg)
-    # metadata.extensions.extension_elements = new_list
-    
-    # ... Piuttosto non devo specificare gli algoritmi di firma/criptazione...
     metadata.extensions = None
 
     # attribute consuming service service name patch
     service_name = metadata.spsso_descriptor.attribute_consuming_service[0].service_name[0]
     service_name.lang = 'it'
     service_name.text = conf._sp_name
-    
+
     ##############
     # avviso 29 v3
```

Comment:
What do you think to put this code in a specialized function?

Reply:
I think it will improve the readability, but again, it wasn't the scope of this PR. I will do it since it costs little effort.

Reply:
@peppelinux Done! 👍🏻
Check it out and resolve the conversation.